src/JuliaRunClient.jl

```diff
@@ -301,6 +301,16 @@ function _type_name_query(ctx::Context, path::String, job::JRunClientJob, query:
     parse_resp(resp)
 end
 
+function initialize(num_workers, ctx=Context(), job=self())
+    initParallel()
+    setJobScale(ctx, job, num_workers)
+    waitForWorkers(num_workers)
+end
```

Comment:
probably return `ctx` from this function?

Reply:
Yes. Will change.